apps/readest-app/src/services/tts/EdgeTTSClient.ts

```diff
@@ -63,64 +63,50 @@ export class EdgeTTSClient implements TTSClient {
     const { marks } = parseSSMLMarks(ssml, this.#primaryLang);
 
     if (preload) {
-      // preload the first 2 marks immediately and the rest in the background
-      const maxImmediate = 2;
-      for (let i = 0; i < Math.min(maxImmediate, marks.length); i++) {
-        const mark = marks[i]!;
+      for (const mark of marks) {
```

Comment:
The cold start of TTS can take longer, especially if the first paragraph is lengthy, since it waits for the entire paragraph to preload.